requirements.txt

```diff
@@ -1,11 +1,3 @@
-astroid==2.0.4
 Django==2.1.3
```

Comment:
Dude! why did u remove everything from requirements ? Those are necessary packages!
First install all the requirements.txt packages in ur virtual environment

Reply:
I restored requirements again